routes/api/proxy.toml

```diff
@@ -19,7 +19,7 @@ handlers = ["proxy.GetNext"]
 [[route]]
 method = "GET"
 group = "proxy"
-httpPath = "/nextList/:prefix/:filename"
+httpPath = "/nextList/:prefix/:filename/:maxKeys"
```

Comment:
may be it better to be arg?

like /nextList/:prefix/:filename?maxkeys=100

Reply:
I could, but isn't this a required argument?

Reply:
If it's required, I prefer it to be par of the URL param. I feel that query parameters should be for optional values.

Reply:
it is required.
but args looks much more logical here...

Reply:
If optional, going to keep as such. Much cleaner to handle and follows the required/optional principles